### PR TITLE
Add 'data munging' or 'data wrangling' in Japanese

### PR DESCRIPTION
**Pull request by TomKellyGenetics**
Migrates changes to 'data wrangling' to separate PR. This term seems to need more detailed discussion than the rest of the Japanese entries.

As discussed in #303 with @baileythegreen the English definition could also be improved. I've added 'data munging' as a duplicate but not sure this is the right way to handle synonyms. 
## Author: 

- @TomKellyGenetics 
- @naoe-tatara 

## Language: 

- Japanese

## Terms defined:

- データマンジング  (data munging)
- データラングリング (data wrangling)

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -1815,10 +1829,17 @@
 
 
 - slug: data_wrangling
+  ref:
+   - data_munging
   en:
     term: "data wrangling"
     def: >
       A colloquial name for small-scale [data engineering](#data_engineering).
```
Comment (TomKellyGenetics) [inline glossary.yml:1837]:
```suggestion
      A colloquial term for the process of organising and preparing data for analysis.
      It is also used to refer to [data engineering](#data_engineering) by domain experts.
```

**Review by TomKellyGenetics — COMMENTED** [review]
The Japanese translation has already been review in #303.

>      データ解析を行う事前準備としてデータを整理、加工することを表す口語表現。
>      比較的小規模な[データーエンジニアリング](#data_engineering)という意味でも使われる。

It differs from the English version in some respects. I've suggested a more literal version of the Japanese below.

**Comment by naoe-tatara** [comment]
Hi, if I understand correctly, the Japanese translation/definition parts are done/approved but English definition of data wrangling needs review/approval?

**Comment by naoe-tatara** [comment]
Hi, I understand that this is not about review of Japanese translation and I am probably not correct person here to review. Is it possible to withdraw as a reviewer here @elletjies ?

**Comment by elletjies** [comment]
Closing this pull request since the last activity took place in April 2021. Please feel free to reopen in the future to make the necessary changes.

**Pull request closed without merging**